Include/modsupport.h

```diff
@@ -56,55 +56,14 @@ PyAPI_FUNC(int) PyModule_ExecDef(PyObject *module, PyModuleDef *def);
 
 #define Py_CLEANUP_SUPPORTED 0x20000
 
+/* The API and ABI versions are left for backwards compatibility.
```

Comment:
Can't we just... update them?

Reply:
I don't think it particularly matters, it could be inferred from `Py_LIMITED_API` if that's set.

It's just an arbitrary value with a purpose that isn't _really_ going away, so deprecating them seems unnecessary when we could just keep using them for their intended purposes (and ignore that they didn't get updated for a large number of ABI changes).

Reply:
Or just use it as one of the values that goes into the struct. We can include more information if needed, it just seems like a waste to declare these names as irrelevant when we could just as easily make them relevant.

Reply:
I've restored the comment.
I'll likely need to move the constants because of `PC\python_ver_rc.h`; that PR can touch the comment.